Standards/scs-XXXX-v1-default-rules-for-security-groups.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Default Rules for Security Groups
+type: Standard
+status: Draft
+track: IaaS
+---
+
+## Introduction
+
+Security Groups in IaaS (OpenStack) are part of the network security mechanisms provided for the users.
+They resemble sets of virtual firewall rules allowing specific network traffic at a port of a VM that connects it to a network.
+They are project-bound, which means that all Security Groups that are newly created are only available to the project in that they were created.
+This is also the case for the default Security Group that is created for each project as soon as the project itself is created.
+
+## Terminology
+
+Security Group (abbr. SG)
+  Set of ip table rules, used for tenant network security.
+
+Security Group Rule (abbr. SG Rule)
+  A single ip table rule, that is part of a SG.
+
+Administrator (abbr. Admin)
+  Operator = User of an OpenStack cloud with the admin role.
+
+## Motivation
+
+The rules of a Security Group can be edited by any user with the member role within a project.
+But when a Security Group is created it automatically incorporates a few SG rules that are configured as default rules.
+Since the 2023.2 release, this set of default rules can be adjusted by administrators[^1][^2].
+In combination with the OpenStack behavior that when a VM is created with no Security Group specified, the default SG of the project is automatically applied to the ports of the VM,
+a user cannot be sure which firewall rules are applied to such a VM.
+
+Therefore this standard proposes default Security Group rules that MUST be set by administrators to avoid differences in default network security in different IaaS environments.
+
+[^1]: [Tracking of development for editable default SG rules](https://bugs.launchpad.net/neutron/+bug/1983053)
+[^2]: [Release Notes of Neutron 2023.2](https://docs.openstack.org/releasenotes/neutron/2023.2.html)
+
+## Design Considerations
+
+Until the 2023.1 release (antelope) the default Security Group rules are hardcoded in the OpenStack code.
```

Comment:
Language nitpicks: Until + present tense reads oddly to me, I suggest to:

- Use `Up to` or `Before` instead of `Until`, or
- Use past tense

(but native speakers are welcome to override this suggestion)